lib/endpoint/tags/table.js

```diff
@@ -126,6 +126,47 @@ function rowToParameter (i, el) {
     params.type = 'objects[]'
   }
 
+  if (params.name === 'content') {
+    if (/reaction type/.test(params.description) === true) {
+      params.enum = ['+1',
+        '-1',
+        'laugh',
+        'confused',
+        'heart',
+        'hooray']
+    }
+  }
+
+  if (params.name === 'type') {
+    if (/Normally this is a `commit`/.test(params.description) === true) {
+      params.enum = ['commit',
+        'tree',
+        'blob']
+    }
+  }
+
+  if (params.name === 'state') {
+    if (/active/.test(params.description) === true) {
```

Comment:
I think just checking for `/active/` is too risky, it could easily cause false positive in future. I’d suggest we check for `/only `"active"` will be accepted/i`